acp/identity/identity_test.go

```diff
@@ -0,0 +1,27 @@
+// Copyright 2024 Democratized Data Foundation
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+package identity
+
+import (
+	"testing"
+
+	"github.com/decred/dcrd/dcrec/secp256k1/v4"
+	"github.com/stretchr/testify/require"
+)
+
+func Test_DIDFromPublicKey_ProducesDIDForPublicKey(t *testing.T) {
+	pubKey := &secp256k1.PublicKey{}
```

Comment:
todo: Please add a test using a `519` key as well

Reply:
Currently the identity struct only seems to support secp256k1 keys. Adding other key types would be outside the scope of this PR